.github/workflows/ci.yml

```diff
@@ -105,7 +105,7 @@ jobs:
 
   win:
     name: Windows x64
-    runs-on: "windows-2019"
+    runs-on: "windows-2022"
```

Comment:
I think we should go straight to the latest 2025 if it works.

Reply:
Sure

Reply:
Looking at the documentation for these configurations, windows-2025 does not have Mercurial anymore. I think it's ok to not verify Mercurial dependent tests on Windows (or perhaps at all), so we could add assumes to those tests and check if Mercurial is available.